packages/engine/Source/Core/IonGeocoderService.js

```diff
@@ -64,6 +105,29 @@ Object.defineProperties(IonGeocoderService.prototype, {
       return undefined;
     },
   },
+  /**
+   * @memberof IonGeocoderService.prototype
+   * @type {IonGeocodeProvider}
+   */
+  geocodeProvider: {
```

Comment:
```suggestion
  geocodeProviderType: {
```

This may be a bit nit-picky, but `type` may help distinguish this from from the `*GeocoderService` instance. I got a bit confused when looking through the specs which was which.